src/components/app/ThemeSelector.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useThemes } from '../../state/CharacterProvider';
+
+const ThemeSelector = () => {
+  const { handleThemeChange } = useThemes();
+ 
+  return (
+    <>
+      <label> Light Theme </label>
+      <input
+        type="radio"
+        name="theme"
+        value="light"
+        onChange={handleThemeChange}/>
+      <label> Dark Theme</label>
+      <input
+        type="radio"
+        name="theme"
+        value="dark"
+        onChange={handleThemeChange}/>
+    </>
```

Comment:
Radio buttons work, could've had a button attached to a boolean as well. Lots of different directions on solving this.